Add vendor blurb to sampling doc (#4945)
open-telemetry/opentelemetry.io: content/en/docs/concepts/sampling/index.md

SDK, you will find support for sampling in the respective documentation pages:

{{% sampling-support-list " " %}}

### Vendors

> Many [vendors](/ecosystem/vendors) offer comprehensive sampling solutions that
> incorporate Head Sampling, Tail Sampling, and other features that can support
> sophisticated sampling needs. If you are sending telemetry to a vendor, it may
> be beneficial to use their sampling solutions.

Reviewer: ```suggestion
Many [vendors](/ecosystem/vendors) offer comprehensive sampling solutions that
incorporate head sampling, tail sampling, and other features that can support
sophisticated sampling needs. If you are sending telemetry to a vendor, consider
using their sampling solutions.
```
